Projects/Social-Trading/SA/Function-Libraries/SocialEntitiesProfile.js

```diff
@@ -473,6 +478,48 @@ exports.newSocialTradingFunctionLibrariesSocialEntitiesProfile = function () {
 
                 SA.projects.network.globals.memory.maps.USER_PROFILES_BY_ID.set(userProfile.id, inMemoryUserProfile)
             }
+            
+            function saveApiAppFile() {
+             
+                const SOCIAL_TRADING_REPO_NAME = profileMessage.socialEntityHandle + "-" + profileMessage.socialEntityType.replace(' ', '-') + "-Data"
+                /*
+                Try to load the Secrets file. 
+                */
+                let filePath = global.env.PATH_TO_SECRETS + '/'
+                let fileName = "ApisSecrets.json"
+                let fileContent 
+                try {
+                    fileContent = SA.nodeModules.fs.readFileSync(filePath + '/' + fileName)
+                }catch(err) {
+                    /* If the file does not exist, then we'll get here.*/
+                }
+                
+                let secretsFile
+                if (fileContent === undefined) {
+                    secretsFile = {
+                        secrets: []
+                    }
+                } else {
+                    secretsFile = JSON.parse(fileContent)
+                }
```

Comment:
This should no longer be needed after the above changes.

Reply:
Needed in line with the above comments.